sdk/nodejs/runtime/invoke.ts

```diff
@@ -383,18 +383,21 @@ function deserializeResponse(
 /**
  * Dynamically calls the function `tok`, which is offered by a provider plugin.
  */
```

Comment:
Maybe we comment this to indicate that it's a helper, and put this comment on `call` with an augmented version on `callSingle`?